cmd/cli/create_secret_basicauth.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2025 Stefan Prodan.
+// SPDX-License-Identifier: AGPL-3.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/fluxcd/pkg/runtime/secrets"
+	"github.com/spf13/cobra"
```

Comment:
```suggestion
	"github.com/spf13/cobra"

	"github.com/fluxcd/pkg/runtime/secrets"
```

Let's do the same across all files. I think linters can help us catch these things?

Reply:
In some controllers we have:

* stdlib
* external
* fluxcd/pkg
* internal